Tasks/UseDotNetV1/task.json

```diff
@@ -0,0 +1,100 @@
+{
+    "id": "B0CE7256-7898-45D3-9CB5-176B752BFEA6",
+    "name": "UseDotNet",
+    "friendlyName": "Use .NET Core",
+    "description": "Acquires a specific version of the .NET Core SDK from the internet or the local cache and adds it to the PATH. Additionally provides basic auth.",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "satisfies": [
+        "DotNetCore"
+    ],
+    "demands": [],
+    "ecosystem": ".NET Core",
+    "instanceNameFormat": "Use .NET Core",
+    "inputs": [
+        {
+            "name": "version",
+            "type": "string",
+            "label": "Version",
+            "defaultValue": "1.0.4",
+            "required": true,
+            "helpMarkDown": "Version Spec of the version to use.  Examples: 10.x, 10.15.1, >=10.15.0"
+        },
+        {
+            "name": "auth",
+            "type": "string",
+            "label": "Auth",
+            "required": false,
+            "helpMarkDown": "The authentication feed/service point to use"
+        },
+        {
+            "name": "proxy",
```

Comment:
arent we removing this for now?

Reply:
Yeah, removed it in the code but forgot to take it out of task.json. Updated appropriately.